include/Decoder.hpp

```diff
@@ -15,49 +15,49 @@
 using json = nlohmann::json;
 
 enum class GrowthVariant {
-    ALL_COMPONENTS, // standard growth
-    INVALID_COMPONENTS,
-    SINGLE_SMALLEST,
-    SINGLE_RANDOM,
-    SINGLE_QUBIT_RANDOM
+    AllComponents, // standard growth
+    InvalidComponents,
+    SingleSmallest,
+    SingleRandom,
+    SingleQubitRandom
 };
 
 [[maybe_unused]] static GrowthVariant growthVariantFromString(const std::string& architecture) {
     if (architecture == "ALL_COMPONENTS" || architecture == "0") {
-        return GrowthVariant::ALL_COMPONENTS;
-    } else if (architecture == "INVALID_COMPONENTS" || architecture == "1") {
-        return GrowthVariant::INVALID_COMPONENTS;
+        return GrowthVariant::AllComponents;
+    } else if (architecture == "INVALID_COMPONENTS" || architecture == "1") { // NOLINT(readability-else-after-return)
```

Comment:
Why not just replace these `else if's with just if's as suggested by the warning?